### PR TITLE
[14.0][MIG] account_invoice_supplierinfo_update_discount: Migration to version 14.0

### PR DESCRIPTION
**Pull request by LudLaf**
@Tecnativa TT38072

### CODE REVIEW

**Comment by LudLaf** [comment]
@ernestotejeda
@cesar-tecnativa
@chienandalu
@pedrobaeza

Please review.

**Review by chienandalu — COMMENTED** [review]
Rebase now that #1216 is merged

**Comment by pedrobaeza** [comment]
/ocabot migration account_invoice_supplierinfo_update_discount

File: account_invoice_supplierinfo_update_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+.. contents::
+   :local:
+
+Known issues / Roadmap
+======================
+
+Odoo ``account`` module displays discounts on invoice lines only if ``sale``
+module is installed. That is bad design. To avoid useless dependencies, this
+module does not depend on sale module, and so displays allways discount
```
Comment (pedrobaeza) [inline account_invoice_supplierinfo_update_discount/README.rst]:
```suggestion
module does not depend on sale module, and so displays always discount
```

**Review by pedrobaeza — APPROVED** [review]

File: account_invoice_supplierinfo_update_discount/README.rst (hunk 1 above)
Comment (chienandalu) [reply account_invoice_supplierinfo_update_discount/README.rst]: Done

**Review by cesar-tecnativa — APPROVED** [review]

**Pull request merged**